package.json

```diff
@@ -41,7 +41,10 @@
 		"watch": "^1.0.2"
 	},
 	"dependencies": {
+		"@types/debug": "^4.1.5",
```

Comment:
This should be in dev deps since it's just a @type. Along with the @types/express